frontend/views/components/ListItem.vue

```diff
@@ -9,9 +9,9 @@ li.c-item
       v-if='icon'
       :class='{ [`icon-${icon}`]: icon }'
     )
+      badge(v-if='badgeCount' type='compact')
```

Comment:
It's missing the badgeCount here as children

Reply:
The 'compact' type doesn't show the count of notifications so I didn't pass it down here. :) Do you think we need to pass down the count as a prop instead of slot?

Reply:
Oh right, it makes sense visually. We must not forget to make it more accessible by adding to the span `aria-label="New {slot} notifications"`, so a user using screen reader knows about it.

Here I think we can not put a slot inside a string as we do with props, but I'm not sure. If we can, cool, otherwise we need to rollback it to a prop. Do this for both types of badge.
Last but not least give it a `role="alert"`. This will turn the element "alive" for screen readers. When the content changes, it lets the user know about it.

Reply:
It's a very informative comment..! :)
let me try using '$slots.default' for the aria-label and rolling back to a prop if it doesn't work.